bin/render-activity-stream-html.js

```diff
@@ -40,7 +40,11 @@ function getStrings(locale, allStrings) {
   const localeFallbacks = [DEFAULT_LOCALE, ...similarLocales, locale];
 
   // Get strings from each locale replacing with those from more desired ones
-  return Object.assign({}, ...localeFallbacks.map(l => allStrings[l]));
+  const desired = Object.assign({}, ...localeFallbacks.map(l => allStrings[l]));
```

Comment:
Is `Object.assign` needed? Won’t it do the same thing as `{...localeFallbacks.map()}`?

Reply:
This is using the spread syntax for passing in multiple object arguments to Object.assign. The spread syntax within an object is expecting a single object. So in this case, we're combining multiple objects with multiple string-key/string-value pairs.

Reply:
`{...array}` does actually work but probably not what you expect ;) The array indices are added to the object as keys with matching values. Kinda like object-ifying an array.